sdks/java/core/src/main/java/org/apache/beam/sdk/testing/TestPipelineOptions.java

```diff
@@ -36,6 +36,14 @@
 
   void setTempRoot(String value);
 
+  /**
+   * An alternative tempRoot that has a bucket-default KMS key set. Used for GCP CMEK integration
+   * tests.
+   */
+  String getTempRootKms();
```

Comment:
We shouldn't define this here since this is really specific to GCP.

Two ways you could solve this:
* You can create an interface called GcpTestPipelineOptions within the gcp extensions package and make sure it gets registered with a PipelineOptionsRegistrar.
* Continue to use tempRoot but make sure the that CMEK related ITs are a different gradle test task (with a CMEK appropriate temp root defined) which filters which tests it runs to only the CMEK ones that need a different temp root while those ITs are excluded from any other IT suite.

Reply:
I could add an attribute like NeedsKmsBucket

Reply:
Attribute added and flag removed. Cleaned up test selection a bit.